gen/bobgen-atlas/driver/atlas.go

```diff
@@ -540,3 +541,69 @@ func (p *driver) getEnums() []drivers.Enum {
 
 	return enums
 }
+
+func (d *driver) indexes(table *schema.Table, colFilter drivers.ColumnFilter) []drivers.Index {
+	var indexes []drivers.Index
+
+	filter := colFilter[d.key(table.Schema.Name, table.Name)]
+	only := filter.Only
+	except := filter.Except
+
+	if table.PrimaryKey != nil && len(table.PrimaryKey.Parts) > 0 {
+		shouldSkip := false
+		cols := make([]string, len(table.PrimaryKey.Parts))
+		for i, part := range table.PrimaryKey.Parts {
+			if part.X != nil || drivers.Skip(part.C.Name, only, except) {
+				shouldSkip = true
+			}
+			cols[i] = part.C.Name
+		}
+		if !shouldSkip {
+			pkName := table.PrimaryKey.Name
+			if pkName == "" {
+				switch d.Dialect() {
+				case "psql":
+					pkName = table.Name + "_pkey"
+				case "mysql":
+					pkName = "PRIMARY"
+				case "sqlite":
+					isCompositePrimaryKey := len(table.PrimaryKey.Parts) > 1
+					isIntegerColumnType := false
+					typ, ok := table.PrimaryKey.Parts[0].C.Type.Type.(*schema.IntegerType)
+					if ok {
+						isIntegerColumnType = typ.T == "integer"
+					}
+					if isCompositePrimaryKey || !isIntegerColumnType {
+						pkName = "sqlite_autoindex_" + table.Name + "_1"
+					}
```

Comment:
When the following conditions are met, SQLite doesn't create an `sqlite_autoindex` on the provided primary key:

```c
if( nTerm==1
 && pCol
 && pCol->eCType==COLTYPE_INTEGER
 && sortOrder!=SQLITE_SO_DESC
){
```

The fragment above is from the SQLite source code:

* `nTerm` is the number of columns supplied to the `PRIMARY KEY`
* `pCol` is the first column
* `pCol->eCType` is the column type. SQLite insists on this being declared as an `integer` and not an `int`. If the column is defined as an `int` then an index will be created.
* `sortOrder` is not really applicable to our use case, as I don't think the Atlas schema allows you to define a `PRIMARY KEY DESC` column (but the SQLite specification does).

Reply:
Can you share a link to this? 🙏🏾 


### Side note

A part of me is considering dropping the Atlas and Prisma drivers. 
Since they can be used to do migrations, perhaps the better flow is `prisma/atlas -> migrate -> bob gen from db`

I am not sure how many people are using these drivers, but I'm conflicted about the cost of maintaining them and if it is worth it.

Reply:
I think I will take them out.

At best, I could keep them alive in a separate repository, but I'm less and less convinced that it should be part of the core.

Kindly hold off on the Prisma implementation. You already did a lot with this one. I really appreciate it.

Reply:
Okay, I'll just fix the linting errors on this one then, and let you decide whether to merge or discard it.

Reply:
Thanks you once more for your amazing contribution 🙏🏾